ONSdigital/eq-survey-runner
# Answer labels not rendered correctly in error page issue

### What is the context of this PR?
Fixes #1538 which was caused by the form object being created from the unrendered block during a POST.

Some other fixes that came out of this work:

- Updated pytest and pytest-cov
- Talisman updated itself so the patched talisman object was removed as this was causing issues
- Minor fixes to tests since flask has been updated
- Integration tests no longer output debug and info logs on failures

### How to review 
- Run the `test_multiple_piping` schema
- Complete survey up to what-is-your-address-group (e.g. question "Does Fred Bloggs live at 44 Norton").
- Without selecting an answer, click continue.
- The error page is displayed
- Verify that the address is still correctly rendered into the first option's label
